components/inputs/documentation/date-time-inputs.md

```diff
@@ -0,0 +1,109 @@
+# Date & Time Inputs
+
+## Date Inputs
+
+The `<d2l-input-date>` component consists of a text input field for typing a date and an attached calendar (`<d2l-calendar>`) dropdown. The dropdown opens on click of the text input, or on enter or down arrow press if the text input is focused. It displays the `value` if one is specified, or a placeholder if not, and reflects the selected value when one is selected in the `calendar` or entered in the text input.
+
+![example screenshot of date input](../screenshots/date.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-date.js';
+</script>
+<d2l-input-date
+  label="Start Date"
+  value="2020-11-20">
+</d2l-input-date>
+```
+
+**Properties:**
+
+- `label` (String, required): accessible label for the input
+- `disabled` (Boolean): disables the input
+- `label-hidden` (Boolean): hides the label visually (moves it to the input's `aria-label` attribute)
+- `value` (String, default: `''`): value of the input. This should be in ISO 8601 calendar date format (`YYYY-MM-DD`) and should be localized to the user's timezone (if applicable).
```

Comment:
Probably a good time to have a section in here on how to convert to and from the user's timezone? Then "localized to the user's timezone" could link down to that?